### PR TITLE
add dbt integration doc

### PR DESCRIPTION
**Pull request by hengm3467**
## Info

- **Description**

 Add a guide about how to transform data in RisingWave using dbt. Also update the integration list.

- **Notes**

  [ Include any supplementary context or references here. ]

- **Related code PR**

  [ Provide a link to the relevant code PR here, if applicable. ]

- **Related doc issue**
  
  Resolves [ Provide a link to the relevant doc issue here, if applicable. ]

## For reviewers

- **Preview**

  [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **upcoming** version of the documentation. ]

- **Key points**

  [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

File: docs/transform/use-dbt.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+- port: 4566
+- user: root
+- database: dev
+
+If you use different RisingWave configurations, please ensure that the configurations here match those in RisingWave.
+
+:::info
+
+If you are familiar with dbt, you can directly clone [this repository](https://github.com/risingwavelabs/dbt_rw_nexmark) to get started. This repository provide a dbt project to manage nexmark queries in a materialized view way.
+
```
Comment (neverchanje) [inline docs/transform/use-dbt.md]: It's not only unnecessary but also confusing to include this line. `dbt_rw_nexmark` is only a demo project. It's not a required step for users to get started.

File: docs/transform/use-dbt.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+## Define dbt models
+
+The dbt models for managing data transformations in RisingWave is similar to typical dbt sql models. The main differences are the materializations. We customized the materializations to fit the data processing model of RisingWave.
+
+RisingWave accepts these four [materializations](https://docs.getdbt.com/docs/build/materializations):
+
+- `table`. Create a table. To use this materialization, add `{{ config(materialized='table') }}` to your model SQL files.
+- `view`. Create a view. To use this materialization, add `{{ config(materialized='view') }}` to your model SQL files.
+- `ephemeral`. Use Common Table Expression instead. To use this materialization, add `{{ config(materialized='ephemeral') }}` to your model SQL files.
```
Comment (neverchanje) [inline docs/transform/use-dbt.md]: " Use Common Table Expression instead. " > "It uses RisingWave's [common table expression](https://www.risingwave.dev/docs/current/query-syntax-with-clause/) under the hood."

File: docs/transform/use-dbt.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+- `materializedview`. Create a materialized view. This materialization is corresponding to the `incremental` one in dbt. Since RisingWave is designed to use materialized view to manage data transformation in an incremental way, you don’t need to use `incremental` models and can just use `materializedview`. To use this materialization, add `{{ config(materialized='materializedview') }}` to your model SQL files.
```
Comment (neverchanje) [inline docs/transform/use-dbt.md]: "Create a [materialized view](https://www.risingwave.dev/docs/current/sql-create-mv/)"

**Review by neverchanje — COMMENTED** [review]
The rest LGTM

**Review by neverchanje — APPROVED** [review]

**Review by chenzl25 — APPROVED** [review]
LGTM!

**Pull request merged**